client/state/data-layer/wpcom/sites/admin-menu/test/index.js

```diff
@@ -60,6 +58,22 @@ describe( 'handlers', () => {
 					},
 				],
 			},
+			{
+				icon: 'dashicons-default',
+				slug: 'my-custom-menu-3',
+				title: 'Home',
+				type: 'menu-item',
+				url: '/home',
+				children: [
+					{
+						parent: 'my-custom-menu-3',
+						slug: 'my-custom-menu-4',
+						title: 'Settings',
+						type: 'submenu-item',
+						url: 'https://example.wordpress.com/wp-admin/settings.php',
+					},
+				],
+			},
 		];
 		const sanitizedMenu = [ ...unsafeMenu ];
 		sanitizedMenu[ 0 ].url = '';
```

Comment:
Add additional test here to check `sanitizedMenu[ 1 ][ 1 ].url` is also cleaned up.

Reply:
`unsafeMenu[ 1 ]` contains safe URLs, so we need to test that they are kept.